pkgs/os-specific/linux/bpftools/default.nix

```diff
@@ -15,7 +15,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-xDalSMcxLOb8WjRyy+rYle749ShB++fHH9jki9/isLo=";
   };
 
-  patches = [ ./strip-binary-name.patch ];
+  patches = [
+    ./strip-binary-name.patch
+  ] ++ lib.optionals stdenv.hostPlatform.isPower64 [
```

Comment:
```suggestion
```
lets make sure this continue to works in the future

Reply:
Done, the patch is applied on all platforms.